### ISSUE TITLE
Need full feature on Nuke Write node.

### ISSUE THREAD

**Issue by MarcoCheng** [PrismPipeline/Prism#7]
Hello, I'm a nuke compositor, I've tried the "Write Prism" node, there are lots of regular write node option missed. 

Included: 
- 32 FULL FLOAT output option (i tried to export STmap within Write Prism, definitely sure it only outputs 16 half float exr. 
- Filepath name without frame hash, such as 'render.v001.exr' instead forced 'render.####.v001.exr' 

**Describe the solution you'd like**
Just copy the regular Nuke write node perhaps.

BTW I really love Prism，Good JOB and thank you for what you did!

**Comment by MarcoCheng** [comment]
Also nice to have add comment option when you create first empty files

**Comment by RichardFrangenberg** [comment]
Yes the WritePrism node writes out 16 bit exrs. It is just a gizmo, which contains a regular write node. You could use the "Copy to group" button on the node tab and open the group with ctrl+enter. Then you could change it to 32 bit if you need it.
I agree that the frame padding is not needed when only one frame will be rendered. The comment for empty files would be useful, too.
I keep track of all requests here:
https://prism-pipeline.com/forum/topic/list-of-feature-ideas/
I added your first point as [0090]. The option for changing the comment is already at [0037]
